### DISCUSSION TITLE
Identity server - supertoken

### DISCUSSION THREAD

**Discussion by antony0901** [nashtech-garage/nodejs-msa#115] [Q&A] [answered]
I am seeing the architecture of the project that is using an identity server that is written in Go. I am wondering why you have not chosen Supertoken (https://supertokens.com/docs/microservice_auth/introduction). It's written in NodeJs so then you will have full NodeJS stack

**Comment by VuiLenDi** [comment, marked as answer]
Super Tokens wrote in Java and needed more features such as goauthentik. We get goauthentik for our example, that's not fit for our architecture. We can use others for our template.
